src/overflow/explainer.md

```diff
@@ -284,6 +284,76 @@ Below, we'll explore some of the features
 that are still missing or difficult
 for authors to get right.
 
+### Carousel example
+
+![Carousel boxes](../carousel-boxes.svg)
+
+As a strawman example,
+a developer should be able to use the following HTML
+to create the visualized carousel:
+
+```html
+<carousel>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+  <li></li>
+</carousel>
+```
+
+The visualized carousel
+can be constructed
+with the following style
+dynamically generating all of the required components:
+
+```css
+carousel {
+  grid-template-columns: [previous] auto [scroller] 1fr [next] auto;
+  grid-template-rows: 
+    [carousel] 1fr 
+    [markers] auto;
+  
+  > li {
+    /* Flow into scroller area */
+    grid-flow: --scroller;
+  }
+  
+  &::grid-flow(scroller) {
```

Comment:
- use dashed ident in both place, to make the connection clear? 
- add a grid-area to position the flow?

```suggestion
  > li {
    /* Flow into scroller grid-flow defined below */
    grid-flow: --scroller;
  }
  
  &::grid-flow(--scroller) {
    grid-area: scroller;
```

Reply:
Done. Thanks for catching that! I also added a grid-area to position the grid-flow for --markers.